smartsim/experiment.py

```diff
@@ -491,10 +493,16 @@ def create_ensemble(
         :return: ``Ensemble`` instance
         :rtype: Ensemble
         """
+        if name is None:
+            raise AttributeError("Entity has no name. Please set name attribute.")
+        check_path = init_default(osp.join(self.exp_path, name), path, str)
+        entity_path: str = osp.abspath(check_path)
+
```

Comment:
Let's try not to use the `init_default` function here. What's wrong with just doing

```
entity_path = osp.join(self.exp_path, name)
```

Reply:
yeah no need for init_default, Ill change